Spell_Chech_Example.c

```diff
@@ -6,7 +6,8 @@
 #include <Elementary.h>
 #include <Efl_Ui.h>
 
-
+Eo * attribute_factory = NULL;
+Eina_List *handles_list = NULL;
```

Comment:
I know it doesn't matter, much in this small example, but just so we encourage good practices around our codebase (examples included!). Please make these static (and the dictionary below, and in general all of them).